7/SiloHost/src/Program.cs

```diff
@@ -67,8 +67,22 @@ public static Task Main()
                     });
                     siloBuilder.ConfigureApplicationParts(applicationPartManager =>
                         applicationPartManager.AddApplicationPart(typeof(HelloWorld).Assembly).WithReferences());
+
+                    /*Registering Feature Management, to allow DI of IFeatureManagerSnapshot in HelloWorld class.
+                     Using built in Percentage filter to demonstrate a feature being on/off.*/
+                    siloBuilder.ConfigureServices(svc =>
```

Comment:
Let's rename `svc` to `serviceCollection` and also not use fluent syntax to make the code look more aligned with what's already here.